Tools/Scripts/webkitdirs.pm

```diff
@@ -2805,7 +2806,10 @@ sub generateBuildSystemFromCMakeProject
 
     push @args, "-DLTO_MODE=$ltoMode" if ltoMode();
 
-    if (isPlayStation()) {
+    if (isAnyWindows()) {
+        push @args, '-DCMAKE_TOOLCHAIN_FILE="' . $ENV{VCPKG_ROOT} . '\\scripts\\buildsystems\\vcpkg.cmake"';
+        push @args, '-DVCPKG_TARGET_TRIPLET=x64-windows-webkit'
```

Comment:
Why do you have to create a new triplet `x64-windows-webkit`? Use built-in triplet `x64-windows`.

Reply:
`vcpkg help triplet` for more info.

Reply:
Its required to replace `zlib` with `zlib-ng` see https://github.com/microsoft/vcpkg/pull/43147